packages/components/src/form/docs/Form.stories.mdx

```diff
@@ -147,7 +147,12 @@ Form fields can be grouped together within fieldsets.
     <Story name="fieldset">
         <Form>
             <Fieldset label="Shipping Address">
-                <Inline>
+                <Grid
+                    templateColumns={{
+                        base: "1fr",
+                        sm: repeat(2, "1fr")
+                    }}
```

Comment:
I still don't get why someone would use a Grid instead of Flex to horizontally align 2 fields :)

Reply:
Either / or. Both achieve the same results, I prefer grid since the repeat and 1fr are explicit and so much easier to use than flex. I could use flex and swap direction too.